src/server/models/cacheable_queries/organization.js

```diff
@@ -18,22 +18,10 @@ const organizationCache = {
     }
     return getConfig("TWILIO_MESSAGE_SERVICE_SID", organization);
   },
-  getMessageServiceAuth: async (organization, contact, messageText) => {
-    // Note organization won't always be available, so we'll need to conditionally look it up based on contact
-    if (messageText && /twilioapitest/.test(messageText)) {
-      return {
-        messagingServiceSid: "fakeSid_MK123",
-        authToken: "foobar",
-        apiKey: "foobarbaz"
-      };
-    }
+  getTwilioAuth: async organization => {
     return {
-      messagingServiceSid: getConfig(
-        "TWILIO_MESSAGE_SERVICE_SID",
-        organization
-      ),
-      authToken: getConfig("TWILIO_API_KEY", organization),
-      apiKey: getConfig("TWILIO_AUTH_TOKEN", organization)
+      authToken: getConfig("TWILIO_AUTH_TOKEN", organization),
```

Comment:
I think what we should maybe do is have different values that are encrypted vs. not encrypted -- so first we should test for `hasConfig("TWILIO_AUTH_DATA_ENCYPRTED")` and if so, then decrypt things here and pass it back (or throw if decryption fails).  If not, then we fallback and do getConfig("TWILIO_AUTH_TOKEN") -- and then it can come from environment variables.